alf/utils/plot_tb_curves.py

```diff
@@ -29,9 +29,22 @@
 
 HOME = os.getenv("HOME")
 
+
 # A namedtuple that represents a curve with mean and upper/lower bounds
-MeanCurve = namedtuple(
-    "MeanCurve", ['x', 'y', 'min_y', 'max_y', 'name'], default_value=())
+class MeanCurve(
+        namedtuple(
+            "MeanCurve", ['x', 'y', 'min_y', 'max_y', 'name'],
+            default_value=())):
+    def normalized_auc(self):
+        """Return the averaged ``y``, ``min_y``, and ``max_y`` over the x-axis.
+        It can also approximate the normalized area under the curve.
+        """
+        return self.final_y(N=len(self.y))
```

Comment:
Why does this approximate normalized AUC?

Reply:
> Why does this approximate normalized AUC?

Assuming x ticks are evenly separated by x_delta. The area under curve can be approximated by \sum_t y[t] * x_delta. Then the normalized AUC is this value divided by the x_delta * N, which is the average of y values.

Reply:
So it is not the normalized auc in your paper? I feel average_y seems to be a more accurate name.

Reply:
> So it is not the normalized auc in your paper? I feel average_y seems to be a more accurate name.

Unless there is official definition of normalized auc, I actually defined this concept in the paper, meaning auc normalized by x range.